frontend/src/services/validate-csv.service.ts

```diff
@@ -74,8 +74,17 @@ export async function validateCsv(
   })
 }
 
+function removeNewLinesFromTables(template: string) {
+  // Get all text within <table> tags
+  return template.replace(/<table\s*>(.*?)<\/table\s*>/gs, (match) =>
```

Comment:
I tried to insert a table like this in template B: 

<table style="width:100%">
  <tr>
    <th>Firstname</th>
    <th>Lastname</th>
    <th>Age</th>
  </tr>
  <tr>
    <td>Jill</td>
    <td>Smith</td>
    <td>50</td>
  </tr>
  <tr>
    <td>Eve</td>
    <td>Jackson</td>
    <td>94</td>
  </tr>
</table>

This is the result: 
<img width="1680" alt="Screenshot 2020-07-24 at 11 20 47 AM" src="https://user-images.githubusercontent.com/33112945/88357995-e2653880-cd9f-11ea-835f-147f9f4166f1.png">

I think when the table has attributes, eg: `table style="width:100%"` the regex doesn't match and the white spaces won't be trimmed

Reply:
ure right